bin/crail

```diff
@@ -70,4 +70,4 @@ fi
 export CLASSPATH="$bin"/../jars/*:"$bin"/../conf:.
 export LD_LIBRARY_PATH="$bin/../lib:$LD_LIBRARY_PATH"
 
-exec "$JAVA" -Dproc_$COMMAND -XX:MaxDirectMemorySize=64G -Dsun.nio.PageAlignDirectMemory=true -Xmn16G $CLASS "$@"
+exec "$JAVA" -Dproc_$COMMAND -Dsun.nio.PageAlignDirectMemory=true $CLASS "$@"
```

Comment:
Can we safely remove those parameters? What about MaxDirectMemorySize, do we need this to support large memory configurations? What about Xmn16G, is removing this not creating a performance problem for large memory configurations?

Reply:
MaxDirectMemorySize is definitely not needed it just sets a limit for direct memory otherwise it will take as much as there is heap space. The Xmx/n is a problem for large clusters but what default value should we set? I can add a crail-env.sh to set these values, but by default I still would not set anything. I can add a warning in the script that suggests setting these values (given the user has lots of memory).